README.md

```diff
@@ -468,3 +470,667 @@ Response:
     }
 }
 ```
+
+10. Filtering by attributes:
+
+You need use attributes query parameter which is an associative array where key is the attribute name and value is an array of attribute values.
+For e.g:
+```php
+$this->client->request('GET', '/shop-api/products', ['attributes' => ['Mug material' => ['Wood']]], [], ['ACCEPT' => 'application/json']);
```

Comment:
If I understand it correctly `Mug material` is an attribute code. If so, I would suggest to rename it to something more obvious like `MUG_MATERIAL_CODE`. Right now, it looks like attribute name.